Dockerfile.bigquery

```diff
@@ -0,0 +1,13 @@
+# Must be run from project root, in order to pick up indexer_util.
```

Comment:
Since the common utils distill down to creating an index, we could also consider making this `create_index` a general Docker image that runs on top of the base Elasticsearch image, and then have each indexer execute only the application-specific indexer logic. The docker compose/user flow would then be `es up > create index/indices > run indexer(s)`.

Reply:
Not sure why we would separate 2 and 3 into separate steps/images. Seems simpler for indexer.py to call util method for 2.

Reply:
A couple of reasons:
* Having Dockerfiles for subdirectories in the top level directory is confusing and more error prone/complex
* Coupling index creation with running the indexer means we need to handle cases like running multiple indexers on the same index, which has non-transparent behavior to the user

Reply:
Discussed offline. It's a good point that current index creation behavior is broken (wrt to running multiple indexers). In my PR to remove index deletion from indexer_util.py, I'll also only create index if it doesn't exist. That, plus adding "delete step" step to deploy README.md, will cover all cases, I believe.

It's unfortunate the Dockerfile has to move up a dir, but I think this is simpler than creating a separate Dockerfile for index initialization.